src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -294,9 +294,12 @@ public class ArgumentsCore
                 ShortName = "size")]
             public int? InputSize;
 
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Source column separator. Options: tab, space, comma, single character", ShortName = "sep")]
+            [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, HelpText = "Source column separator. Options: tab, space, comma, single character", ShortName = "sep")]
             public string Separator = "tab";
 
+            [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly, HelpText = "Source column separator. Option: single character ONLY", ShortName = "del")]
```

Comment:
>Option: single character ONLY [](start = 147, length = 29)

I think the fact that it's a single character is implied by the fact that the argument is of type `char`. We don't usually clarify that non-array arguments can only have a single value specified, since that information is part of the type anyway.

Reply:
Though it would be nice if this argument was as capable as `Separator`, which can support multiple delimiters.

---
In reply to: [188001984](https://github.com/dotnet/machinelearning/pull/142#discussion_r188001984) [](ancestors = 188001984)

Reply:
When you shift to multiple delimiters, take the same care as is taken with `Separator` to ensure uniqueness, etc.

---
In reply to: [188006909](https://github.com/dotnet/machinelearning/pull/142#discussion_r188006909) [](ancestors = 188006909,188001984)